doc/sphinx/addendum/implicit-coercions.rst

```diff
@@ -414,6 +386,23 @@ coercion path between ``A`` and ``Funclass`` if it exists.
   Check (b 0).
   Unset Printing Coercions.
 
+.. _example-identity-coercion:
+
+We give now an example using identity coercions.
```

Comment:
Perhaps:

```suggestion
.. example:: Identity coercions
```
and indent the entire example, the preferred styling for examples.  This will make it look like this:

![image](https://user-images.githubusercontent.com/1253341/158255468-4b42a84b-d074-4347-8b25-540bd485a4e8.png)

OTOH None of the examples in the chapter use `.. example::`, so up to you.  If you're willing to adjust all the examples in the chapter, that would be great.